src/App.tsx

```diff
@@ -1,26 +1,175 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState, useRef } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { createTodo, deleteTodo, getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { ErrorMes } from './components/ErrorMes';
+import { FilterName } from './types/enumFilterName';
+import { Error } from './types/enumError';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<Error | ''>('');
```

Comment:
`''` - it's not a type. 
use ur enum for it - `Error.Default = ''`

```suggestion
  const [errorMessage, setErrorMessage] = useState<Error>(Error.Default);
```